src/main/kotlin/pt/up/fe/ni/website/backend/util/MultipartFile.kt

```diff
@@ -0,0 +1,7 @@
+package pt.up.fe.ni.website.backend.util
+
+import org.springframework.web.multipart.MultipartFile
+
+fun MultipartFile.filenameExtension(): String? {
+    return this.originalFilename?.substringAfterLast(".") ?: ""
+}
```

Comment:
What this returns is not nullable, right? I'd either change to `String` or remove `?: ""`. Maybe the second is more predictable

Reply:
I think extensions do make sense for a utils package, but we should explain that better. I suggest changing the name to `MultipartFileExtensions` or creating an `extensions` package inside `utils`. Perhaps it's a good idea to go w/ the second approach and also move `annotations` to `utils`. That way, utils is just an aggregation of many utility packages with meaningful names. If you think `utils/` seems too underwhelming for such a task, I've seen `pkg/` used in a lot of GO projects

Reply:
Done

Reply:
Please check my first comment again, seems weird to return "" by default when the type is nullable